### PR TITLE
Fix up Labs and add settings UI

### PR DESCRIPTION
**Pull request by lexisother**
Need I say more?

### CODE REVIEW

**Comment by lexisother** [comment]
For the reviewers, just click this 
image

File: src/Powercord/coremods/registry.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
-  require('./badges'),
-  require('./utility-classes'),
-  require('./react-devtools')
-];
+module.exports = [
+  require('./no-track'),
+  require('./router'),
+  require('./store'),
+  require('./badges'),
+  require('./utility-classes'),
```
Comment (asportnoy) [inline src/Powercord/coremods/registry.js:6]: Should make this dynamically load

**Review by asportnoy — CHANGES_REQUESTED** [review]
You forgot to remove the one at the bottom of the general settings page

File: src/Powercord/coremods/registry.js (hunk 1 above)
Comment (lexisother) [reply src/Powercord/coremods/registry.js:6]: That's not the scope of the PR.

**Comment by lexisother** [comment]
> You forgot to remove the one at the bottom of the general settings page

I'll get rid of that one, apologies!

**Review by asportnoy — COMMENTED** [review]
Just give this another rebase to include the linting changes, then once that passes I will merge.

**Comment by lexisother** [comment]
Seems like some commits failed the rebase, so it made new ones... I love Git!

**Review by asportnoy — APPROVED** [review]

**Pull request merged**